contracts/ckb-cheque-script/src/entry/helper.rs

```diff
@@ -0,0 +1,39 @@
+use ckb_std::{
+    ckb_constants::Source,
+    ckb_types::{packed::*, prelude::*},
+    high_level::{load_cell, load_input, QueryIter},
+};
+use alloc::vec::Vec;
+use super::hash;
+
+pub fn has_input_by_lock_hash(lock_hash: [u8; 20]) -> bool {
+    QueryIter::new(load_cell, Source::Input)
```

Comment:
```suggestion
    QueryIter::new(load_cell_lock_hash, Source::Input)
```
是否可以用这个方法 ？另外传递进来的 lock_hash 是 20 bytes, load_cell_lock_hash 是 32 bytes, 比较的时候需要截取下前 20 bytes 把？

Reply:
No syscalls function `load_cell_lock_hash `!